### PR TITLE
 Install and use explicit typescript and npm versions 

### PR DESCRIPTION
**Pull request by arielshaqed**
Otherwise local dev machine versions are used, which is unpredictable.

### CODE REVIEW

File: src/output.ts [hunk 1]
```diff
@@ -36,13 +37,20 @@ const tsconfig: Record<Runtime, any> = {
   [Runtime.node]: tsconfigBase,
 };
 
+function pathTo(module: string, binary: string) {
+  return require.resolve(join(module, 'bin', binary));
```
Comment (bergundy) [inline src/output.ts]: It's not necessarily in `bin`, it just happens to be the in the case of these 2 modules.

**Review by bergundy — APPROVED** [review]
Thanks,
I approve but I'd appreciate if you address the single comment.

File: src/output.ts (hunk 1 above)
Comment (arielshaqed) [reply src/output.ts]: Agreed.  Removed to make it smoother.

**Review by arielshaqed — COMMENTED** [review]
Thanks; PTAL!

**Pull request merged**